### PR TITLE
Remove commbadge waffle and old emails (bug 1099355)

### PR DESCRIPTION
**Pull request by eviljeff**
Mainly rewriting tests that depended on the old emails.

### CODE REVIEW

File: mkt/reviewers/tests/test_views.py [hunk 1]
```diff
@@ -1300,21 +1301,39 @@ def test_public_sign_failure(self, sign_mock, json_mock,
 
 
 class TestReviewMixin(object):
+    # E.g commreply+12e0caffc4ca4174a6f62300c0ff180a@marketplace.firefox.com .
+    COMM_REPLY_RE = r'^commreply\+[a-f0-9]+\@marketplace\.firefox\.com$'
```
Comment (robhudson) [inline mkt/reviewers/tests/test_views.py]: you can use `re.compile()` to compile this so each time it is used it doesn't have to do it again.

File: mkt/reviewers/tests/test_views.py [hunk 2]
```diff
@@ -1509,6 +1521,8 @@ def test_incomplete_cant_approve(self):
         eq_(self.get_app().status, mkt.STATUS_NULL)
 
     def test_notification_email_translation(self):
+        # https://bugzilla.mozilla.org/show_bug.cgi?id=1127790
+        raise SkipTest
```
Comment (robhudson) [inline mkt/reviewers/tests/test_views.py]: :cry:

File: mkt/developers/utils.py [hunk 3]
```diff
@@ -8,7 +8,6 @@
 from django.template.defaultfilters import filesizeformat
```
Comment (robhudson) [inline mkt/developers/utils.py]: `reverse` no longer used.

File: mkt/developers/utils.py [hunk 4]
```diff
@@ -17,7 +16,6 @@
 from mkt.comm.utils import create_comm_note
 from mkt.constants import APP_PREVIEW_MINIMUMS, comm, PRERELEASE_PERMISSIONS
 from mkt.reviewers.models import EscalationQueue
-from mkt.reviewers.utils import send_reviewer_mail
 from mkt.site.helpers import absolutify
```
Comment (robhudson) [inline mkt/developers/utils.py]: absolutify no longer used

File: mkt/comm/views.py [hunk 5]
```diff
@@ -4,7 +4,6 @@
 from django.db.models import Q
```
Comment (robhudson) [inline mkt/comm/views.py]: `'Q' imported but unused`

File: mkt/comm/views.py [hunk 6]
```diff
@@ -4,7 +4,6 @@
...
 from django.shortcuts import get_object_or_404
 
-import waffle
 from rest_framework import status
 from rest_framework.authentication import BaseAuthentication
 from rest_framework.decorators import (api_view, authentication_classes,
```
Comment (robhudson) [inline mkt/comm/views.py]: `'BooleanField' imported but unused`
`'BaseFilterBackend' imported but unused`

File: mkt/reviewers/models.py [hunk 7]
```diff
@@ -6,12 +6,10 @@
 from django.db.models import Sum
```
Comment (robhudson) [inline mkt/reviewers/models.py]: `'settings' imported but unused`

File: mkt/reviewers/utils.py [hunk 8]
```diff
@@ -10,7 +10,6 @@
 from django.utils.datastructures import SortedDict
```
Comment (robhudson) [inline mkt/reviewers/utils.py]: 'reverse' imported but unused
'translation' imported but unused
'absolutify' imported but unused
'to_language' imported but unused

File: mkt/reviewers/tests/test_views.py [hunk 9]
```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 import json
 import os
+import re
 import time
 from datetime import datetime, timedelta
 from itertools import cycle
```
Comment (robhudson) [inline mkt/reviewers/tests/test_views.py]: 'Group' imported but unused
GroupUser' imported but unused

**Comment by robhudson** [comment]
This looks fine to me. I may want to give it another pass just b/c it's a lot of changing and I glossed over on parts of it.

**Comment by ngokevin** [comment]
I think we can remove reviewers/includes/review_history.html . Still planning on switching over to do direct DB queries rather than API calls later, but been busy with frontend UI changes. Glossed over, looks good.

File: mkt/reviewers/tests/test_views.py (hunk 1 above)
Comment (eviljeff) [inline mkt/reviewers/tests/test_views.py]: I can, but I read in the docs https://docs.python.org/2/library/re.html#module-contents that the regex's are cached by match() automatically.

**Comment by eviljeff** [comment]
removed unused imports; removed review_history.html

**Comment by robhudson** [comment]
r+. Needs a rebase.

**Pull request merged**